NOTES.md

```diff
@@ -1,3 +1,143 @@
+# Release 2.2.0 (Modern Times)
+
+Released on 2015-11-XX
+
+Release highlights:
+
+* Atomic changefeeds
+* Performance & Scalability improvements
+
+Read the [blog post][2.2-release] for more details.
+
+[2.2-release]: http://rethinkdb.com/blog/2.2-release/
+
+## Compatibility ##
+
+Data files from RethinkDB versions 1.16.0 onward will be automatically migrated.
+As with any major release, back up your data files before performing the upgrade.
+
+If you're upgrading from RethinkDB 1.14.x or 1.15.x, you need to migrate your secondary
+indexes first. You can do this by following these steps:
+* install RethinkDB 2.0.5
```

Comment:
These should be sentences:

- Install RethinkDB 2.0.5.
- ...
- Run `rethinkdb index-rebuild`.